### DISCUSSION TITLE
Problem with Levi-Civita

### DISCUSSION THREAD

**Discussion by gekatsia** [FeynCalc/feyncalc#164] [Q&A]
Hi, 
I am using  Feyncalc 9.3.1 (stable version) in Mathematica 13.0 in a windows laptop. 
I am trying to simplify expressions that contain gamma matrices contracted with the Levi -Civita tensor, but Mathematica does not cancel terms such as the one given below (a,b are dummy indices)

`(DiracMatrix[a] Eps[m, n, r, a] - DiracMatrix[b] Eps[m, n, r, b]) // 
  Contract // DiracReduce`
I have used Feyncalc in the past, and older versions could 'see' that the above expression evaluates to zero.  
Now I have tried a couple of things, but nothing seems to work.
I would appreciate any suggestions.
Thank you.

**Comment by vsht** [comment]
Hi,

- `DiracMatrix` is deprecated, please use [GA](https://feyncalc.github.io/FeynCalcBookDev/GA.html) instead.
- `Eps` is an internal symbol and the notation `Eps[a,b,c,d]` is not supported anymore. The correct shortcut is called [LC](https://feyncalc.github.io/FeynCalcBookDev/LC.html).

To simplify such expressions one needs to canonicalize the dummy indices:
```
(GA[a] LC[m, n, r, a] - GA[b] LC[m, n, r, b]) // FCCanonicalizeDummyIndices
```

**Reply by gekatsia** [reply]
Thank you very much!!
